### ISSUE TITLE
Training CogVideoX-2B controlnet but generating noise image

### ISSUE THREAD

**Issue by lubinBoooos** [TheDenk/cogvideox-controlnet#30]
(1) I pre-generate the vae embedding and prompt embeding on OpenVid-1M (partial videos, about 10000 videos)
(2) I train the controlnet by shell script below, with 1 epoch and 4 GPU of 24G
# shell script
export MODEL_PATH="/DataDisk2/LLMs/CogVideoX-2B"
export PYTORCH_CUDA_ALLOC_CONF=expandable_segments:True
export CUDA_VISIBLE_DEVICES=0,1,2,3

accelerate launch --config_file accelerate_config_machine_single.yaml --multi_gpu \
  train_controlnet_2.py \
  --tracker_name "cogvideox-controlnet" \
  --gradient_checkpointing \
  --pretrained_model_name_or_path $MODEL_PATH \
  --enable_tiling \
  --enable_slicing \
  --seed 42 \
  --mixed_precision fp16 \
  --output_dir "outputs" \
  --height 480 \
  --width 720 \
  --fps 8 \
  --max_num_frames 49 \
  --video_root_dir "/DataDisk2/datasets/cogvideoX2B_openvid1M_data" \
  --stride_min 1 \
  --stride_max 3 \
  --hflip_p 0.5 \
  --controlnet_transformer_num_layers 8 \
  --controlnet_input_channels 3 \
  --downscale_coef 8 \
  --controlnet_weights 1.0 \
  --init_from_transformer \
  --train_batch_size 1 \
  --dataloader_num_workers 0 \
  --num_train_epochs 10 \
  --checkpointing_steps 1000 \
  --gradient_accumulation_steps 2 \
  --learning_rate 1e-5 \
  --lr_scheduler cosine_with_restarts \
  --lr_warmup_steps 250 \
  --lr_num_cycles 1 \
  --enable_slicing \
  --enable_tiling \
  --optimizer AdamW \
  --adam_beta1 0.9 \
  --adam_beta2 0.95 \
  --max_grad_norm 1.0 \
  --allow_tf32 

# Porblem:
I load the checkpoint with 10000 steps trainning, but I cannot get the right generation result,. When I using the pretrained controlnet model to initlizing the trainable parameters, I can get a good model.  Do I need more train epochs or more data?

**Comment by TheDenk** [comment]
Hi, yep, you need more data and more epochs. Try about 40k video and 500-1k A100 gpu hours.

**Comment by lubinBoooos** [comment]
Yes，thanks for your response, I get the right genenration result when I enhanced the data and trained more epoches. And I also change the configuration
# shell script
--learning_rate 1e-4
--train_batch_size 2
--controlnet_weights 1.0
